templates/compute/resources/vkcs_compute_volume_attach.md.tmpl

```diff
@@ -11,20 +11,7 @@ description: |-
 {{ .Description }}
 
 ## Example Usage
-### Basic attachment of a single volume to a single instance
-{{tffile "templates/compute/resources/vkcs_compute_volume_attach/basic-single-single/main.tf"}}
-
-### Attaching multiple volumes to a single instance
-{{tffile "templates/compute/resources/vkcs_compute_volume_attach/multi-single/main.tf"}}
-
-Note that the above example will not guarantee that the volumes are attached in
```

Comment:
well, this is important

Reply:
i added example to add volumes one after another